raster/r.univar/r.univar_main.c

```diff
@@ -133,8 +133,8 @@ int main(int argc, char *argv[])
     /* table field separator */
     zone_info.sep = G_option_to_separator(param.separator);
 
-    zone_info.min = 0.0 / 0.0;	/* set to nan as default */
-    zone_info.max = 0.0 / 0.0;	/* set to nan as default */
+    zone_info.min = NAN; /* set to nan as default */
```

Comment:
This change is not Cmake related and should be separated out

(into a new NAN related PR)